Dockerfile

```diff
@@ -1,23 +1,24 @@
 ARG 	CHEMISTRY_PACKAGE=jchem
-FROM maven:3-openjdk-8 as builder
+FROM maven:3-openjdk-11 as builder
 
 FROM 	builder as dependencies
 ARG     CHEMISTRY_PACKAGE
 ENV     CHEMISTRY_PACKAGE=${CHEMISTRY_PACKAGE}
 
 FROM 	dependencies as jchem
-ADD 	lib/jchem-16.4.25.0.jar /lib/jchem-16.4.25.0.jar
-RUN     mvn install:install-file -Dfile=/lib/jchem-16.4.25.0.jar -DartifactId=jchem -DgroupId=com.chemaxon -Dversion=16.4.25.0 -Dpackaging=jar -DgeneratePom=true -DcreateChecksum=true
+# ADD 	lib/jchem-16.4.25.0.jar /lib/jchem-16.4.25.0.jar
+# RUN     mvn install:install-file -Dfile=/lib/jchem-16.4.25.0.jar -DartifactId=jchem -DgroupId=com.chemaxon -Dversion=16.4.25.0 -Dpackaging=jar -DgeneratePom=true -DcreateChecksum=true
 
 FROM 	${CHEMISTRY_PACKAGE} as compile
+ADD settings.xml /root/.m2/settings.xml
 ADD 	pom.xml /src/pom.xml
 WORKDIR /src
-RUN 	mvn dependency:resolve -P ${CHEMISTRY_PACKAGE}
+RUN 	mvn dependency:resolve -P ${CHEMISTRY_PACKAGE} -X
```

Comment:
I don't think we need the maven debug flag on permanently. Can this be removed once you've finished debugging?